### ISSUE TITLE
Intellisense doesn't work with conflicts with legacy styles

### ISSUE THREAD

**Issue by mbuena-ninety** [wkillerud/vscode-scss#85]
- VS Code version: 1.85.0
- Extension version: v2.14.5
- Operating system: macOS Sonoma 14.0

Reproducible Case:

Steps to Reproduce:

1. Entry file `_ds.scss`
```
@forward '../core/colors/colors';
```
3. Colors file `_colors.scss`
```
@function myColor($color) {

}
```

4. In my component file
```
@use '../../design-system/ds' as ds;

.my-component {
     color: ds.myColor()
}
```

In our app we have a bunch of other files that have legacy styles. We created a new folder that's pretty bare bones that just has the new colors as shown in the setup above.
However doing `ds.` doesn't trigger the intellisense.

Upon inspection, I'm seeing a lot of errors in the Some Sass Ouput tab that seems to be related to all the old/existing legacy styles:
```
You can't have more than one poster comment.
Annotation `parameter` is not allowed on comment from type `css`.
Annotation `parameter` is not allowed on comment from type `css`.
Parser for annotation `breaking` not found.
Parser for annotation `breaking` not found.
Annotation `return` is not allowed on comment from type `mixin`.
Parser for annotation `deprecation` not found.
@return must at least have a type. Location: undefined:84:87
@return must at least have a type. Location: undefined:105:110
@return must at least have a type. Location: undefined:115:120
@return must at least have a type. Location: undefined:141:144
@return must at least have a type. Location: undefined:149:156
@return must at least have a type. Location: undefined:166:174
Item `declaration` refers to `css.declaration` from type `mixin` but this item doesn't exist.
Item `declaration` refers to `configuration` from type `mixin` but this item doesn't exist.
Item `validate-theme` refers to `validate-theme-styles to validate only theme keys.` from type `function` but this item doesn't exist.
Item `validate-theme-styles` refers to `validate-theme to validate both theme keys and theme values.` from type `function` but this item doesn't exist.
Item `_validate-theme-values` refers to `Use `validate-theme()` to validate both theme keys and theme values.` from type `function` but this item doesn't exist.
Item `add-link` refers to ``add-link()`` from type `mixin` but this item doesn't exist.
@return must at least have a type. Location: undefined:89:98
@return must at least have a type. Location: undefined:327:339
Parser for annotation `examples` not found.
@return must at least have a type. Location: undefined:273:333
Parser for annotation `breaking` not found.
Parser for annotation `breaking` not found.
Parser for annotation `breaking` not found.
Parser for annotation `breaking` not found.
Parser for annotation `breaking` not found.
Parser for annotation `breaking` not found.
Parser for annotation `breaking` not found.
Parser for annotation `breaking` not found.
Parser for annotation `breaking` not found.
Parser for annotation `breaking` not found.
Parser for annotation `breaking` not found.
Parser for annotation `breaking` not found.
Parser for annotation `breaking` not found.
Parser for annotation `breaking` not found.
```

**Comment by wkillerud** [comment]
Hello.

The errors you're seeing indicate problems with the SassDoc in your code, either:

- Some format error, like the messages about [`@return`](http://sassdoc.com/annotations/#return)
- Or a type of annotation that's not part of the [official spec](http://sassdoc.com/annotations/) (`breaking`).

The errors should have no effect, other than perhaps not giving you the expected result for docs on hover and such.

Just checking, the new styles compile OK? No broken links?

Can you try excluding the folders with legacy code using the _Scanner Exclude_ setting, and see if that helps?

https://github.com/wkillerud/vscode-scss#exclude-files-or-folders

![](https://github.com/wkillerud/vscode-scss/assets/1223410/302fb564-7d78-4cf7-9559-0c665ab5b11f)

**Comment by mbuena-ninety** [comment]
I appreciate your quick response! Currently we're not using any SassDocs in our own code, but I was able to find that those SassDocs errors were actually coming from Angular Material. In our old styles we are doing `@use '@angular/material' as mat;` which I assume was causing it to get included.

I ended up adding the paths to all our legacy styles just to be safe which seems to get rid of the errors.
![Screenshot 2023-12-13 at 12 27 11 PM](https://github.com/wkillerud/vscode-scss/assets/146779849/6d4502e7-47ad-4e62-ad89-66f2cad2af75)

As seen in the screenshot above, I've also checked the "Suggest From Use Only" setting. If I'm understanding it correctly it should only suggest from the one path I'm using below:
![Screenshot 2023-12-13 at 12 31 19 PM](https://github.com/wkillerud/vscode-scss/assets/146779849/0917fbe1-97e8-4f94-9a6f-b0ae93dee224)

However I'm still not getting suggestions to pop up.
